classes/submission/SubmissionAgencyDAO.inc.php

```diff
@@ -47,16 +47,17 @@ function getLocaleFieldNames() {
 	 * Get agencies for a specified submission ID.
 	 * @param $submissionId int
 	 * @param $locales array
+	 * @param $version
 	 * @return array
 	 */
-	function getAgencies($submissionId, $locales) {
+	function getAgencies($submissionId, $locales, $version = null) {
 
 		$returner = array();
 		foreach ($locales as $locale) {
 			$returner[$locale] = array();
 			$agencies = $this->build($submissionId);
 			$submissionAgencyEntryDao = DAORegistry::getDAO('SubmissionAgencyEntryDAO');
-			$submissionAgencies = $submissionAgencyEntryDao->getByControlledVocabId($agencies->getId());
+			$submissionAgencies = $submissionAgencyEntryDao->getByControlledVocabId($agencies->getId(), null, $version);
```

Comment:
Why does getByControlledVocabId in SubmissionAgencyEntryDAO has the parameter version and in ControlledVocabEntryDAO not?